### DISCUSSION TITLE
Persistent "Get A Registration Key!" banner in toolbar

### DISCUSSION THREAD

**Discussion by Eternal9100** [iterate-ch/cyberduck#16335] [Q&A] [answered]
I have two CD and an MD Licence file, I keep getting a red banner on the toolbar even though th licence is applied and CD exits without a nag dialogue. I have scrubbed and reinstalled CD and MD  with new installers and with advised reboots for MD. I recovered another licence file from the website help section for CD and MD. The banner clears when the licence is applied but returns on the next CD session. MD is working normally. Please see screenshot:
![cyberduck-nag](https://github.com/user-attachments/assets/efa095ac-d9c4-498a-8f59-05492a450283)

Any advice welcome. It only an minor irritation. Everything seems to work.

E9100

**Comment by dkocher** [comment, marked as answer]
Please update to version 9.0.3 on _Beta_ which has the issue [^1] resolved.

[^1]: iterate-ch/cyberduck#16329

**Reply by Eternal9100** [reply]
Thanks. Spend the additional E10 wisely! I may be going all Linux next year because having run W11 in a VM, er... Non. I'll miss CD and MD.
